source/index.js

```diff
@@ -1,14 +1,6 @@
-import { getLoginTarget, getLoginTargets } from "./loginTargets.js";
 import { getSharedObserver } from "./UnloadObserver.js";
 
 // Initialise the DOM unload observer
 getSharedObserver();
 
-/**
- * The Locust library
- * @module Locust
- */
-export default {
-    getLoginTarget,
-    getLoginTargets
-};
+export { getLoginTarget, getLoginTargets } from "./loginTargets";
```

Comment:
I think that this is a breaking change btw.

But it probably makes it for the better since before you had to do `const Locust = require('@buttercup/locust').default` so now the readme documentation is probably right 👍